code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm

```diff
@@ -176,6 +176,12 @@
 /proc/_list_swap(list/L, Index1, Index2)
 	L.Swap(Index1, Index2)
 
+/proc/_text2path(text)
+	return text2path(text)
+
+/proc/_path2text(path)
+	return "[path]"
```

Comment:
This seems unecessary

Reply:
Intentional to explicitly convert to string